RecoTracker/TkTrackingRegions/src/RectangularEtaPhiTrackingRegion.cc

```diff
@@ -47,6 +47,34 @@ namespace {
 using namespace PixelRecoUtilities;
 using namespace std;
 
+std::vector<bool> RectangularEtaPhiTrackingRegion::checkTracks(reco::TrackCollection const& InputCollection) const {
+  std::vector<bool> mask(InputCollection.size(), false);
+  int it = -1;
+  math::XYZPoint regOrigin(origin().x(), origin().y(), origin().z());
+  for (auto const& track : InputCollection) {
+    it++;
+    if (track.pt() < ptMin()) {
+      continue;
+    }
+
+    if (!etaRange().inside(track.eta())) {
```

Comment:
if I understand the original code correctly, this (and the next) check should be applied only if `theUseEtaPhi == true` ?

Reply:
OK, not, that parameter does something different ...